v03_pipeline/var/vep_config/vep-GRCh37.json

```diff
@@ -0,0 +1,20 @@
+{"command": [
+    "/vep",
+    "GRCh37",
+    "--format", "vcf",
+    "__OUTPUT_FORMAT_FLAG__",
+    "--everything",
+    "--allele_number",
+    "--no_stats",
+    "--cache", "--offline",
+    "--minimal",
+    "--assembly", "GRCh37",
+    "--fasta", "/opt/vep/.vep/homo_sapiens/Homo_sapiens.GRCh37.dna.primary_assembly.fa.gz",
+    "--plugin", "LoF,human_ancestor_fa:/opt/vep/.vep/loftee_data/GRCh37/human_ancestor.fa.gz,filter_position:0.05,min_intron_size:15,conservation_file:/opt/vep/.vep/loftee_data/GRCh37/phylocsf_gerp.sql,gerp_file:/opt/vep/.vep/loftee_data/GRCh37/GERP_scores.final.sorted.txt.gz",
```

Comment:
The loftee_data for `GRCh37` is nested but it isn't nested for `38`.  So I think this is right as is.